tests/lib/ocpp/common/test_message_queue.cpp

```diff
@@ -2,14 +2,25 @@
 #include <gtest/gtest.h>
```

Comment:
Missing license header
```suggestion
// SPDX-License-Identifier: Apache-2.0
// Copyright 2020 - 2023 Pionix GmbH and Contributors to EVerest
#include <gtest/gtest.h>
```